packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -28,7 +28,7 @@ import '../runner/flutter_command.dart';
 import '../tester/flutter_tester.dart';
 import '../vmservice.dart';
 
-const String protocolVersion = '0.3.0';
+const String protocolVersion = '0.4.0';
```

Comment:
Technically not a breaking change, just adding API.

Reply:
I think on an earlier PR we changed from adding 0.0.1 to 0.1.0 because it was a new feature, so I though this was better safe than sorry (though I suspect I'm the only person doing anything with this number right now). LMK if you want me to change it though; I'm not shipping anything relying on it for at least a few weeks.

Reply:
No, IntelliJ isn't using the number - no need to change it back. Technically, removing API would be breaking, adding to existing is generally not.